submissions/akulahere/index.html

```diff
@@ -0,0 +1,245 @@
+<!doctype html>
+<head>
+    <meta charset="utf-8">
+    <title>Interstellar</title>
+    <meta name="description" content="Interstellar booking">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<header>
+    <a href="#" class="step-button">&#8249;</a>
+    <h1 class="header-name">
+        Select your places
+    </h1>
+    <a href="#" class="step-button">&#8250;</a>
+</header>
+<main class="main-container">
+    <section class="info-container">
+        <h2 class="info-item">Interstellar</h2>
+        <p class="info-item">
+            <time datetime="2001-05-15 19:00">15 may 19:00</time>
+            .
+        </p>
+        </p>
+        <p class="info-item">Near future, the population of the Earth is on the verge of extinction. Hoping to find a
+            new habitable planet, a group of explorers undertakes the most important mission in the history of the
+            humankind — they travel beyond the bounds of our galaxy through a space-time tunnel — a ‘black hole.’ The
+            journey has one aim — to learn, whether the humankind has a chance for salvation and future.
+        </p>
+    </section>
+    <section class="cinema-scheme">
+        <div class="screen-line"></div>
+        <div class="screen-capture">
```

Comment:
Add some styles for `border-top`:
![image](https://user-images.githubusercontent.com/16196199/107272478-6ccc6b80-6a56-11eb-8754-17601da0335d.png)